### DISCUSSION TITLE
Oculus Quest 2 - Controller Button Access

### DISCUSSION THREAD

**Discussion by loibnerg** [microsoft/MixedRealityToolkit-Unity#11206] [Q&A] [answered]
Dear community,

I'm very sorry for asking such a trivial question but how can I detect a button input of an oculus quest 2 touch controller in an unity script? The mixed reality toolkit hand tracking sample is up and running and all works fine. I just don't know how to program this.

Thank you all for your help!

**Comment by MaxWang-MS** [comment, marked as answer]
Please take a look at this Unity doc: https://docs.unity3d.com/Packages/com.unity.inputsystem@1.4/manual/QuickStartGuide.html
